core/tx_pool.go

```diff
@@ -636,7 +636,7 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 		return ErrInsufficientFunds
 	}
 	// Ensure the transaction has more gas than the basic tx fee.
-	intrGas, err := IntrinsicGas(tx.Data(), tx.AccessList(), tx.To() == nil, true, pool.istanbul)
+	intrGas, err := IntrinsicGas(tx.Data(), tx.AccessList(), tx.To() == nil, true, pool.istanbul, true)
```

Comment:
Here TxPool is being conservative and always expects initcode charge being covered by transaction.

Not sure yet how would we check her for extra EIP being activated and whether it's worth it.

Reply:
Maybe it's better to change it to always assume EIP-3860 is **not** activated, then nothing changes for the tx pool right now, but with EIP activated it will not drop some txs that don't have enough gas to pay for initcode.

Which would be easier to solve when EIP is accepted to the fork, and not is an "extra EIP".